### PR TITLE
Ignore: Throttling sample

### PR DESCRIPTION
**Pull request by j82w**
# Pull Request Template
## Description

Sample getting the retry after time for a throttled response.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: Microsoft.Azure.Cosmos.Samples/Usage/ItemManagement/Program.cs [hunk 1]
```diff
@@ -199,6 +202,50 @@ private static async Task<SalesOrder> CreateItemsAsync()
...
+            {
+                deleteTasks.Add(container.DeleteItemStreamAsync(order.Id, new PartitionKey(order.AccountNumber)));
+            }
+
+            await Task.WhenAll(deleteTasks);
+
+            foreach(Task<ResponseMessage> taskMessage in deleteTasks)
+            {
+                ResponseMessage response = await taskMessage;
+                if(response.StatusCode == HttpStatusCode.TooManyRequests)
```
Comment (bchong95) [inline Microsoft.Azure.Cosmos.Samples/Usage/ItemManagement/Program.cs:235]: hmm ... if only there was a `TooManyRequestsResponseMessage` that exposes the `RetryAfter` as a `TimeSpan` :)

Comment (j82w) [reply Microsoft.Azure.Cosmos.Samples/Usage/ItemManagement/Program.cs:235]: That doesn't work unless we have the user cast to a type based on the error which seems unreasonable.

File: Microsoft.Azure.Cosmos.Samples/Usage/ItemManagement/Program.cs [hunk 2]
```diff
@@ -199,6 +202,50 @@ private static async Task<SalesOrder> CreateItemsAsync()
...
+            for(int i = 1; i <= 1000; i++)
+            {
+                SalesOrder saleOrder = GetSalesOrderSample(Guid.NewGuid().ToString(), accountNumber);
+                salesOrders.Add(saleOrder);
+                batch.CreateItemStream(Program.ToStream<SalesOrder>(saleOrder));
+
+                if(i%100 == 0)
+                {
+                    await batch.ExecuteAsync();
+                }
```
Comment (kirankumarkolli) [inline Microsoft.Azure.Cosmos.Samples/Usage/ItemManagement/Program.cs:221]: batch object re-create?

File: Microsoft.Azure.Cosmos.Samples/Usage/ItemManagement/Program.cs [hunk 3]
```diff
@@ -199,6 +202,50 @@ private static async Task<SalesOrder> CreateItemsAsync()
         }
         // </CreateItemsAsync>
 
+        private static async Task ThrottleItemsAsync()
```
Comment (kirankumarkolli) [inline Microsoft.Azure.Cosmos.Samples/Usage/ItemManagement/Program.cs:205]: Most of the sample code is setup and ids-tracting from main line usage.

Do code docs cover the expected usage?

Comment (kirankumarkolli) [reply Microsoft.Azure.Cosmos.Samples/Usage/ItemManagement/Program.cs:205]: 429 & 449 both will have retrywith. docs should cover both.

Comment (j82w) [reply Microsoft.Azure.Cosmos.Samples/Usage/ItemManagement/Program.cs:205]: I was doing this just to validate it worked based on this issue https://github.com/Azure/azure-cosmos-dotnet-v3/issues/1374. I'm not planning on merging this.

**Pull request closed without merging**